app/frontend/src/cljs/teet/ui/file_upload.cljs

```diff
@@ -148,8 +151,17 @@
       (fn [i ^js/File file]
         ^{:key i}
         [ListItem {}
-         [ListItemText {:primary (.-name file)
-                        :secondary (str (.-type file) " " (format/file-size (.-size file)))}]
+         (.log js/console file)
+         (let [invalidfile? (not (s/valid? :document/file-object file))
+               file (document-model/file-info file)]
+           [ListItemText (merge
+                           {:primary (:file/name file)
+                            :secondary (r/as-component [:<>
+                                                        [:span (merge {} (when invalidfile?
+                                                                           {:style {:color theme-colors/error}}))
+                                                         (str (:file/type file)) (when invalidfile?
+                                                                                   (str " " (tr [:document :wrong-file-type])))]
```

Comment:
other causes of invalidity may be misreported as "wrong file type" here, especially if we add size checks later

Reply:
Changed the check to only check for filetype now. And if we add check for size need to add different error style.